core/src/test/scala/org/apache/spark/sql/delta/HiveDeltaDDLSuite.scala

```diff
@@ -26,11 +26,6 @@ abstract class HiveDeltaDDLSuiteBase
   extends DeltaDDLTestBase {
   import testImplicits._
 
-  override protected def verifyDescribeTable(tblName: String): Unit = {
```

Comment:
why is this removed? not used any more?

Reply:
Sorry should've added a note here. This is the same in all child suites so I moved the definition to `DeltaDDLTestBase` [here](https://github.com/delta-io/delta/pull/1720/files#diff-90717098e565cbbb91601b34e60bc6aca86c6f9ee6e6fd47b5dac75c735a0841L96) and removed them from the subclasses.

Reply:
nice!